source/fundamentals/enterprise-auth.txt

```diff
@@ -1,27 +1,186 @@
+.. _golang-enterprise-authentication-mechanisms:
+
 ====================================
 Enterprise Authentication Mechanisms
 ====================================
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-In this guide, we show you how to authenticate with MongoDB using each
-**authentication mechanism** available exclusively in the MongoDB Enterprise
-Edition.
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available in the MongoDB Enterprise Edition.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
 
-You can use the following mechanisms with the latest version of the MongoDB
-Enterprise Edition:
+You can use the following authentication mechanisms with the latest version of
+the MongoDB Enterprise Edition.
 
-Kerberos (GSSAPI)
-LDAP (PLAIN)
+- :ref:`GSSAPI/Kerberos <golang-kerberos>`
+- :ref:`LDAP (Plain) <golang-LDAP>`
 
-To authenticate using another mechanism, see
-:ref:`golang-authentication-mechanisms`. To learn more about 
-establishing a connection to your MongoDB cluster, see 
+To authenticate using another mechanism, see the
+:ref:`<golang-authentication-mechanisms>` fundamentals page. For
+more information on establishing a connection to your MongoDB cluster, see the
 :ref:`golang-connection-guide`.
 
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using the following methods:
+
+- Using a **connection string**, which is a string
```

Comment:
Suggestion: I think we refer to this as the Connection URI in the Connection Guide, so it could be more clear if used consistently. I think we could also defer the explanation to the Connection Guide.

Comment applies to all instances of "connection string".